src/main/webapp/entries.js

```diff
@@ -42,8 +42,8 @@ module.exports = {
   "project-new": "./resources/js/pages/projects/projects-new.js",
   "project-settings-basic":
     "./resources/js/pages/projects/settings/project-settings-basic.js",
-  "project-settings-remote":
-    "./resources/js/pages/projects/settings/project-settings-remote.js",
+  "project-remote":
+    "./resources/js/pages/projects/remote/index.js",
```

Comment:
I don't think you even need to add the `/index.js`

Reply:
Removed in bb34052964426a9d5f05269592dbeb56afcf1e82